src/PharoLauncher-CLI-Tests/PhLTemplateListCliCommandTest.class.st

```diff
@@ -0,0 +1,34 @@
+Class {
+	#name : #PhLTemplateListCliCommandTest,
+	#superclass : #PhLTemplatesCliCommandTest,
+	#category : #'PharoLauncher-CLI-Tests'
+}
+
+{ #category : #tests }
+PhLTemplateListCliCommandTest >> testListTemplateShouldSucceed [
+	self createOfficialRepositoryInFileSystemWithTemplatesInIt. 
```

Comment:
rename #createOfficialRepositoryInFileSystemWithTemplatesInIt to #createOfficialTemplatesCategoryWithFakeTemplate ?